profile/README.md

```diff
@@ -1,4 +1,76 @@
 # Special Interest Group for Physical AI
 
-This is an organization for the OSRA TGC Special Interest Group for Physical AI
+## 👋 Welcome! We're making ROS the platform of choice for the next
 
+generation of AI-powered robots
+
+### Quick Links
+
+- [Working Groups](docs/working-groups/) - Explore our five focus areas
```

Comment:
This file will render in the GitHub UI so I think we'll need to port this to be full links not relative.